src/MicrosoftTeams.ts

```diff
@@ -449,7 +489,8 @@ namespace microsoftTeams
         {
             ensureInitialized(frameContexts.authentication);
 
-            sendMessage("authentication.authenticate.success", result);
+            localStorage.setItem("authentication.success", result);
```

Comment:
Yes, the auth completion endpoint will have to be in the same domain as the endpoint that kicked off the auth flow. I will make sure to document this before asking developers to switch to it. At this point I don't know of a way to make this work across domains without message posting which depends on window.opener. Once auth completes the tab content/config page would be able to use navigateCrossDomain to move to a different domain if it wants (and that's what VSTS does today).